core/trigProducers.fcl

```diff
@@ -774,6 +774,17 @@ TrigCstProducers : {
       ComboHitCollection: "TTmakePH"
       TestFlag: false
       module_type: "SimpleTimeCluster"
+      HitSelectionBits  : ["EnergySelection"]
```

Comment:
The pattern we encourage is that this fille should be written as:

```
CstSimpleTimeCluster: {
   @table::SimpleTimeCluster 
   # Optionally override some parameters.
}
```

I think this is what you meant by "inherit from" in https://github.com/Mu2e/Offline/pull/1275#issuecomment-2138482604 .

Reply:
Thanks.